solarforecastarbiter/io/reference_observations/srml.py

```diff
@@ -154,7 +162,16 @@ def fetch(api, site, start, end):
     # adjust power from watts to megawatts
     for column in power_columns:
         all_period_data[column] = all_period_data[column] / 1000000
-    all_period_data = all_period_data[var_columns]
+    all_period_data = all_period_data.loc[start:end, var_columns]
```

Comment:
Is this the ValueError @wholmgren mentioned (or KeyError)? Either way, instead of raising, I would prefer catching and logging like line 154 to avoid problems with data ingestion job

Reply:
https://github.com/SolarArbiter/solarforecastarbiter-core/pull/572/files#diff-de89b40a63393dc73c01aa714c77e271R154-R157

Reply:
that's all I meant

Reply:
@wholmgren  I accepted the comment without double checking, the `ValueError` is not raised, but a warning is logged and an empty data frame is returned. So I've removed the Raises docstring I added.

@alorenzo175 I think the only expected Exception here would be the `TypeError` if start/end don't have the same timezone above. I don't believe there are any exceptions that would crop up here cron run to cron run and cause a crash. Is there a potential issue you see here?